src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1596,10 +1596,10 @@
             "enum": ["en", "fr"],
             "default": "en",
             "title": "User Interface Language:",
-            "description": "The IDE's user-interface language."
+            "description": "The IDE''s user-interface language."
         },
         "native_file_dialogs": {
-            "description": "Whether RStudio Desktop will use the operating system's native File and Message dialog boxes.",
+            "description": "Whether RStudio Desktop will use the operating system''s native File and Message dialog boxes.",
```

Comment:
Putting double-apostrophes in the user-prefs-schema.json is causing double-aprostrophes to show up in the UI. For example, I built this branch and see this in the tooltip:

![Screen Shot 2022-05-31 at 4 55 02 PM](https://user-images.githubusercontent.com/10569626/171297280-b393beb3-e0a7-46a9-a584-66ffb722c698.png)

Reply:
ahh I see. I will have this reverted in a bit!